src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -137,12 +143,6 @@ public static final boolean powerOfTwoUsingBits(int numberToCheck) {
         multiDigits.put(90,"ninety");
     }
 
-    private static final int BILLION = 1000000000;
-    private static final int MILLION = 1000000;
-    private static final int THOUSAND = 1000;
-    private static final int HUNDRED = 100;
-    private static final int TEN = 10;
-
```

Comment:
Removing this block addresses the problem that they were not at the beginning of the class.